src/Avalonia.Diagnostics/DevTools.xaml.cs

```diff
@@ -73,7 +78,7 @@ void PreviewKeyDown(object sender, KeyEventArgs e)
                 RoutingStrategies.Tunnel);
         }
 
-        private static void OpenDevTools(TopLevel control)
+        internal static void OpenDevTools(TopLevel control)
```

Comment:
`Attach` is public, shouldn't this be public as well?

Reply:
public api is on line 32, to be consistent with attach api, this is in different class, so didnt want to change that to be public